unitary/examples/tictactoe/tic_tac_toe.py

```diff
@@ -227,9 +229,113 @@ def sample(self, count: int = 1) -> List[str]:
         """
         return [_result_to_str(result) for result in self.board.peek(count=count)]
 
-    def print(self) -> str:
+
+class GameInterface:
+    """
+    A class that provides a command-line interface to play Quantum Tic Tac Toe.
+
+    Initialize by providing an instance of a TicTacToe game, then call play()
+    to run the game.
+
+    Args:
+        game: A TicTacToe instance for the game interface to wrap.
+        file:  Optional IOBase file object to write output to.
+            This enables the battle to write status to a file or string
+            for testing.
+
+    """
+    def __init__(self, game: TicTacToe, file: io.IOBase = sys.stdout):
+        self.game = game
+        self.file = file
+        self.player = "X"
+        self.player_quit = False
+
+    def get_move(self) -> str: 
+        """
+        Gets and returns the player's move.
+
+        Basically a wrapper around input to facilitate testing.
+        """
+        return input(f"Player {self.player} to move (\"help\" for help): ")
+    
+    def player_move(self) -> None:
+        """
+        Interprets the player's move and takes the appropriate action.
+
+        A move can be a one or two letter string within the set [abcdefghi],
+        in which case this function hands the move off to the TicTacToe instance,
+        or one of the GameMoves enums (GameMoves.MAP, GameMoves.EXIT, GameMoves.HELP),
+        which prevent the game loop from alternating to the next player.
+
+        Raises:
+            ValueError if it is still the player's move.
+        """
+        move = self.get_move()
+        print(move)
```

Comment:
Should this be `print(move, file=self.file)`

Reply:
No, that was a testing artifact.